### ISSUE TITLE
IBM x3650 M4 servers report incorrect DRAM size

### ISSUE THREAD

**Issue by ghost** [joyent/smartos-live#374]
From @pannon:

I believe this needs to be re-opened - please see below.

The issue is affecting IBM x3650 M4 servers, we have two of these and both of them report incorrect memory sizes. Both boxes are based on Xeon E5-2609. Older M3 and M2 servers are unaffected - we have tested these.

Here is what was tested so far with the results:
1. Firmware updated to latest on both servers
2. All memory was taken out and only 16GB memory installed (2x 8GB modules) 
3. Downloaded and installed latest SmartOS - memory correctly showing 16GB
4. Downloaded OmniOS (Illumos), booted live mode - memory correctly showing 16GB
5. Redeployed SDC compute node - memory showing Incorrect 1548 MB DRAM only!

In a nutshell all other Illumos based operating systems recognize the full amount of memory - except SDC based compute nodes.

I am happy to do further testing on this, as we have more IBM servers coming for SDC (these will be M5 versions).
